pkgs/by-name/id/ida-free/package.nix

```diff
@@ -16,51 +15,25 @@
   libsForQt5,
   libunwind,
   libxkbcommon,
-  makeDesktopItem,
   makeWrapper,
   openssl,
   stdenv,
   xorg,
   zlib,
 }:
 
-let
-  srcs = builtins.fromJSON (builtins.readFile ./srcs.json);
-in
 stdenv.mkDerivation rec {
   pname = "ida-free";
-  version = "8.4.240320";
+  version = "9.0sp1";
 
-  src = fetchurl {
-    inherit (srcs.${stdenv.system} or (throw "Unsupported system: ${stdenv.hostPlatform.system}"))
-      urls
-      sha256
-      ;
+  src = requireFile {
```

Comment:
Why not put this on the web archive? Having a `requireFile` means a breaking change here, which I'd like not to have.